client/httplib/httplib.go

```diff
@@ -275,6 +276,17 @@ func (b *BeegoHTTPRequest) PostFile(formname, filename string) *BeegoHTTPRequest
 	return b
 }
 
+// PostForm submit formdata like "name1=value1&name2=value2" and encode
+func (b *BeegoHTTPRequest) PostForm(fromdata string) (*BeegoHTTPRequest, error) {
```

Comment:
I think maybe users want to use `map` as the `formdata`.
If we declare `formdata`, I guess that users need to build the string in most cases. Maybe using `map` would be better.